codice/ddf
# [2.26.x] Support startIndex in GetFeature requests on WFS 1.1.0

#### What does this PR do?
Adds configuration to enable `startIndex` and deep paging on `GetFeature` requests for WFS 1.1 servers that support it like Geoserver.

#### Who is reviewing it? 

@derekwilhelm 

#### Select relevant component teams: 


#### Ask 2 committers to review/merge the PR and tag them here.

@jlcsmith 

#### How should this be tested?


- Setup Geoserver with test data with over 1000 results. 
- Configure a WFS 1.1.0 source for that Geoserver. 
- Set the `supportsStartIndex` config option. 
- Verify you can page past 1000 results.
- Disable the `supportsStartIndex` config option. 
- Verify you can page up to 1000 results but fails past 1000.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
